infra/lib/storage-builder.ts

```diff
@@ -29,7 +30,7 @@ export interface storageResources {
         metadataStorageTable: dynamodb.Table;
     };
 }
-export function storageResourcesBuilder(scope: Construct): storageResources {
+export function storageResourcesBuilder(scope: Construct, staging_bucket?: string): storageResources {
```

Comment:
Not a blocker at all since we only have a couple formal parameters here, wondering if `staging_bucket` should be part of an input struct to match the style of most cdk function calls. Definitely can do it later if more parameters materialize.